bee-runtime/Cargo.toml

```diff
@@ -14,9 +14,9 @@ homepage = "https://www.iota.org"
 bee-storage = { version = "0.2.0-alpha", path = "../bee-storage/bee-storage" }
 
 async-trait = "0.1"
-dashmap = "3.11"
+dashmap = "4.0"
```

Comment:
do we still need dashmap in the node?

Reply:
Maybe we'll refactor the event bus to get rid of it, but outside of the scope of this PR